project/cmake/scripts/rbpi/ArchSetup.cmake

```diff
@@ -18,6 +18,7 @@ else()
     set(ARCH arm-linux-gnueabihf)
```

Comment:
Never underestimate some users ability to turn every possible option on ;)
I'd simply force it off here, to be on the save side